CalibFormats/CaloObjects/src/CaloSamples.cc

```diff
@@ -3,14 +3,13 @@
 #include <math.h>
 #include <iostream>
 
-const int CaloSamples::MAXSAMPLES;
-
```

Comment:
@kpedro88 since you don't use the address of `CaloSamples::MAXSAMPLES` apparently, it is fine to remove it. But it would be more safe to keep this line, I think.